Modules/termios.c

```diff
@@ -315,6 +315,141 @@ termios_tcflow_impl(PyObject *module, int fd, int action)
     Py_RETURN_NONE;
 }
 
+/*[clinic input]
+termios.tcgetwinsize
+
+    fd: fildes
+    /
+
+Get the tty winsize for file descriptor fd.
+
+Returns a list [ws_row, ws_col].
+[clinic start generated code]*/
+
+static PyObject *
+termios_tcgetwinsize_impl(PyObject *module, int fd)
+/*[clinic end generated code: output=31825977d5325fb6 input=c7ed8aa957d108c0]*/
+{
+#if defined(TIOCGWINSZ)
+    termiosmodulestate *state = PyModule_GetState(module);
+    struct winsize w;
+    if (ioctl(fd, TIOCGWINSZ, &w) == -1) {
+        return PyErr_SetFromErrno(state->TermiosError);
+    }
+
+    PyObject *v;
+    if (!(v = PyList_New(2))) {
```

Comment:
When returning a constant number of values, it is more typical to use a tuple than a list.  PyTuple_New and equivalent item setting calls.  also update the docstrings in the comment above and on the next function to mention "two item tuple" and use ().

Reply:
@gpshead Sir, thank you for the review. I should mention that I used `PyList_New()` instead of `PyTuple_New()` because I wanted this function to resemble the other low level function `termios.tcgetattr()`, which also returns a list. Do you confirm this change?

Reply:
I ended up making the changes!